decidim/decidim
# Fix organization multitenant issue with meeting's reminders

#### :tophat: What? Why?
This PR fixes a scenario where CloseMeetingReminder is sending Meeting Reminders to any admin user in any organization.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #12502

#### Testing

1. Create 2 organizations inside a single installation 
2. Invite admins 
3. In one organziation create an official meeting 
4. Allow the event to end ( or set the end date in the past ) 
5. Send the reminders 
6. See both admin users receives the notification 
7. Apply patch
8. Repeat 3,4,5 
9. See only the organization admin in which the meeting has been created receives the notification. 

### :camera: Screenshots
*Please add screenshots of the changes you are proposing*
![Description](URL)

:hearts: Thank you!
